DancingGoat/Helpers/Extensions/HtmlHelperExtensions.cs

```diff
@@ -1,37 +1,41 @@
-﻿using System;
-using System.Configuration;
-using System.Linq;
-using System.Linq.Expressions;
-using System.Web.Mvc;
-using System.Web.Mvc.Html;
-using DancingGoat.Areas.Admin;
-using DancingGoat.Models;
-
+﻿using DancingGoat.Models;
 using KenticoCloud.ContentManagement.Helpers.Models;
-using KenticoCloud.Delivery;
 using KenticoCloud.Delivery.ImageTransformation;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.IO;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Net.Http;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
 
 namespace DancingGoat.Helpers.Extensions
 {
     public static class HtmlHelperExtensions
     {
+
         /// <summary>
         /// Generates an IMG tag for an image file.
         /// </summary>
+        /// <remarks>Should replace it with the asp.net core's img TagHelper in each view</remarks>
         /// <param name="htmlHelper">HTML helper.</param>
         /// <param name="asset">Asset</param>
         /// <param name="title">Title</param>
         /// <param name="cssClass">CSS class</param>
         /// <param name="width">Optional width size</param>
         /// <param name="height">Optional height size</param>
         /// <param name="sizes">Media conditions mapping screen width to image size</param>
-        public static MvcHtmlString AssetImage(this HtmlHelper htmlHelper, Asset asset, string title = null, string cssClass = "", int? width = null, int? height = null, ResponsiveImageSizes sizes = null)
+        public static IHtmlContent AssetImage(this IHtmlHelper htmlHelper, IConfiguration configuration, KenticoCloud.Delivery.Asset asset, string title = null, string cssClass = "", int? width = null, int? height = null, ResponsiveImageSizes sizes = null)
```

Comment:
you should use this in views...e.g. article.cshtml
otherwise, pictures will be broken

Reply:
Hmm yes, I know, it was before migrate these extensions methods to the new project, I have to investigate where I don't make use of them.

I will try adding localisation, according to docs, from scratch.

For (4), maybe one of the extension methods is broken.

For (5), I will see what is used there.

And, summarising, I will run the 2 apps, framework vs core, to see what is missing.

Reply:
perfect!